common/daemon.c

```diff
@@ -146,6 +147,12 @@ void daemon_setup(const char *argv0,
 		memleak_init();
 #endif
 
+	/* We rely on libsodium for some of the crypto stuff, so we'd better
+	 * not start if it cannot do its job correctly. */
+	if (sodium_init() == -1)
+		errx(1, "Could not initialize libsodium. Maybe not enough entropy"
+		         " available ?");
```

Comment:
Only a question: What does this("not enough entropy") mean? It means the machine can't run `clightning` forever?

Reply:
No it means that, in this rare case (maybe just after booting), the machine does not have enough source of randomness. https://en.wikipedia.org/wiki/Entropy_(computing)